validate.py

```diff
@@ -117,6 +118,12 @@ def determine_if_mergeable():
   for user, state in sorted(reviews.items()):
     print ('  %s: %s' % (user, state))
 
+  # As the PR gets older, require more and more rejections to "veto" it,
+  # overriding any non-unanimous majority approvals.
+  # Thus, a newly opened PR can be blocked with a single veto (0 days + 1)
+  # A PR that has been open for 3 days requires four rejections to be vetoed.
+  rejections_required_to_veto = math.floor(days_since_last_commit() + 1)
```

Comment:
`days_since_last_commit` isn't a good way to measure PR age.  I can submit a new PR with an old commit on it, and it will read as old.  The functionality you need is in #37